Mathlib/Data/Int/Lemmas.lean

```diff
@@ -77,6 +77,22 @@ theorem natAbs_inj_of_nonpos_of_nonneg {a b : ℤ} (ha : a ≤ 0) (hb : 0 ≤ b)
   simpa only [Int.natAbs_neg] using natAbs_inj_of_nonneg_of_nonneg (neg_nonneg_of_nonpos ha) hb
 #align int.nat_abs_inj_of_nonpos_of_nonneg Int.natAbs_inj_of_nonpos_of_nonneg
 
+/-- A specialization of `abs_sub_le_of_nonneg_of_le` for working with the signed subtraction
+  of natural numbers. -/
+theorem natAbs_coe_sub_coe_le_of_le {a b n : ℕ} (a_le_n : a ≤ n) (b_le_n : b ≤ n) :
+    natAbs (↑b - ↑a) ≤ n := by
```

Comment:
Using up-arrows rather than type ascriptions can confuse lean (though probably never here); this is perhaps more readable and robust:
```suggestion
    natAbs (b - a : ℤ) ≤ n := by
```